misk-hibernate/src/main/kotlin/misk/hibernate/ReflectionQueryFactory.kt

```diff
@@ -499,7 +515,9 @@ internal class ReflectionQuery<T : DbEntity<T>>(
         val constraint = function.findAnnotation<Constraint>()
         val select = function.findAnnotation<Select>()
         val order = function.findAnnotation<Order>()
+        val fetch = function.findAnnotation<Fetch>()
 
+        // TODO (peckb1) should `fetch` and `order` go here?
```

Comment:
When `@Order` was added it was not added to this constraint check. 

(a) Are we allowed to have a method annotated with both `@Order` and `@Select` or was it overlooked?
(b) My assumption is that Fetch should be a solo annotation and should be added into this list regardless of `order`